src/lib/gundb/FeedStorage.js

```diff
@@ -568,12 +569,9 @@ export class FeedStorage {
             // if yes - upload it and store CID instead
             let value = _value
 
-            if (Config.nftLazyUpload && 'avatar' === field && isValidBase64Image(value)) {
-              const avatar = asImageRecord(value)
-              const smallAvatar = { ...avatar }
-
+            if (Config.nftLazyUpload && 'avatar' === field) {
               // keep old base64 value if upload failed
-              value = await UserAvatarStorage.storeAvatars(avatar, smallAvatar).catch(() => _value)
+              value = await updateFeedEventAvatar(_value)
```

Comment:
this needs explanation and doc.
what's the purpose? I see it loads CID then resave to ipfs... seems redundant. Isnt this just to upload old gun base64 to ipfs?